capstan.go

```diff
@@ -505,14 +505,19 @@ func main() {
 						cli.BoolFlag{Name: "pull-missing, p", Usage: "attempt to pull packages missing from a local repository"},
 						cli.StringFlag{Name: "boot", Usage: "specify config_set name to boot unikernel with"},
 						cli.BoolFlag{Name: "verbose, v", Usage: "verbose mode"},
+						cli.BoolFlag{Name: "remote", Usage: "set when previewing the compose-remote"},
 					},
 					Action: func(c *cli.Context) error {
 						repo := util.NewRepo(c.GlobalString("u"))
 						packageDir, _ := os.Getwd()
 
 						pullMissing := c.Bool("pull-missing")
+						implicitRequire := []string{"osv.bootstrap"}
```

Comment:
Is there a benefit of having business logic here instead of inside the `CollectPackage`? Wouldn't passing `remote` flag into this function be better than setting the implicit bootstrap here?